### ISSUE TITLE
Video: when video ends, play button does not have correct size

### ISSUE THREAD

**Issue by inesdgomes** [international-labour-organization/designsystem#523]
Once the video finishes, the play button shows in a weird size. It should have the same size as before the video is played.

<img width="888" alt="video-play-button" src="https://github.com/international-labour-organization/designsystem/assets/22772102/a9cce15f-6412-4b7c-81e2-5590142b4dd1">

**Comment by GGKapanadze** [comment]
@inesdgomes I tried to reproduce the issue but was unable to do that

I tested both react(youtube included) and twig versions

| Browser | Desktop | Mobile |
|--------|--------|--------|
|Chrome | ✅ | ✅ |
| Firefox | ✅ | ✅ |
| Safari | ✅ | ✅ | 

### Some screencasts
https://github.com/international-labour-organization/designsystem/assets/36326203/ddab2b36-5f31-4aec-b59f-00028f09abec


https://github.com/international-labour-organization/designsystem/assets/36326203/b6d62e29-d406-4d3f-a57b-ab5e5d838dbd


https://github.com/international-labour-organization/designsystem/assets/36326203/cb762bf3-3887-4e17-a7a7-bc94b6e75b0a


https://github.com/international-labour-organization/designsystem/assets/36326203/45258be7-7b02-45f9-80dd-a173a917ca03

**Comment by inesdgomes** [comment]
@GGKapanadze @bashlk That's odd. I saw this issue on the staging website where we're using the design system. Could you test the video on this page: https://iloorgstg.prod.acquia-sites.com/meetings-and-events/generation-trying-recapture-lost-youth 

Here's what I see when the video finishes.

https://github.com/international-labour-organization/designsystem/assets/22772102/d88f01d5-ef19-4e1b-a236-7e1da6186b9f

My browser details: http://supportdetails.net/MI896033

**Comment by GGKapanadze** [comment]
@inesdgomes Yes the issue is there on the staging website but the current implementation in the component library overrides the "big" play button while the video is paused but styles aren't applied in Drupal.

There are two options:
1. Drupal is using an outdated version 
2. Drupal re-bundles CSS in a different way so styles aren't generated correctly 

I'll investigate both versions

> Component Library

![image](https://github.com/international-labour-organization/designsystem/assets/36326203/d1212732-f203-4c1d-b6e4-e5ddb083e058)

> Staging Website 

![image](https://github.com/international-labour-organization/designsystem/assets/36326203/58b3fdb9-694a-4449-8df3-5d6a5901feb2)

@bashlk @justintemps

**Comment by justintemps** [comment]
Hey @inesdgomes, here is the current version of the Video Player (it's not on the staging version of the website yet). Can you confirm that this issue is fixed here so we can close this ticket? https://ilo-ui-twig-develop.netlify.app/?path=/story/components-media-video--video

**Comment by inesdgomes** [comment]
@justintemps Yes, it's fixed.

**Comment by justintemps** [comment]
Nope it's not! @GGKapanadze to try to duplicate this on a local Drupal instance to see if the issue isn't coming from the website.
